addons/crm_iap_enrich/models/crm_lead.py

```diff
@@ -41,6 +37,15 @@ def _iap_enrich_leads_cron(self):
         ])
         leads.iap_enrich(from_cron=True)
 
+    @api.model_create_multi
+    def create(self, vals_list):
+        leads = super(Lead, self).create(vals_list)
+        enrich_mode = self.env['ir.config_parameter'].sudo().get_param('crm.iap.lead.enrich.setting', 'manual')
+        if enrich_mode == 'auto':
+            cron = self.env.ref('crm_iap_enrich.ir_cron_lead_enrichment')
```

Comment:
Could add "raise_if_not_found=False" and check that cron exists before calling "_trigger" on it.
Would make it safer if the cron is removed for some reason.